src/App.tsx

```diff
@@ -1,27 +1,64 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { get5First, getAll, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [isSendRequest, setIsSendRequest] = useState(false);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  useEffect(() => {
+    if (isSendRequest) {
+      getAll()
+        .then((good) => {
+          setGoods(good);
+          setIsSendRequest(false);
+        });
+    }
+  }, [isSendRequest]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handlerAllGoods = () => {
+    getAll().then(user => setGoods(user));
```

Comment:
please check other places 
```suggestion
    getAll().then(goods => setGoods(goods));
```